TROUBLESHOOTING.md

```diff
@@ -0,0 +1,98 @@
+# Troubleshooting
+
+## I can't install terraform v0.15.4+
+
+Install terraform:
```

Comment:
This is more than just installing terraform. It's installing a tf version manager that will in turn allow you to install more recent versions of tf than brew will allow on MacOS.

Reply:
Fixed! I stole your words :)